pyathena/sqlalchemy_athena.py

```diff
@@ -398,6 +475,14 @@ def get_view_names(self, connection, schema=None, **kw):
         tables = self._get_tables(connection, schema, **kw)
         return [t.name for t in tables if t.table_type == "VIRTUAL_VIEW"]
 
+    def get_table_comment(self, connection, table_name, schema=None, **kw):
+        metadata = self._get_table(connection, table_name, schema=schema, **kw)
+        return {"text": metadata.comment}
+
+    def get_table_options(self, connection, table_name, schema=None, **kw):
+        metadata = self._get_table(connection, table_name, schema=schema, **kw)
+        return {"awsathena_location": metadata.location}
```

Comment:
The compresssion setting is also required.

Reply:
I'd really like to not rush this. And since this is not really supported, as of yet, there is no regression here.
Considering the [complexity of compression](https://docs.aws.amazon.com/athena/latest/ug/compression-formats.html) in Athena. Adding to that the fact that the storage format can be represented in different way and that compression heavily depends on it. I'd like to do it right, in due time.

If we rush, we will likely expose API we will have a hard time removing later on, if we realize we got it wrong in the first place. I already regret that we added the "compression" dialect kwargs.

FYI as you can see compression information can be stored in different place in the metadata depending on whether the file format handles it internally or it is dealt with externally. Here with a table stored in JSON:
```yaml
TableMetadata:
  Columns: []
  CreateTime: null
  LastAccessTime: null
  Name: json_table
  Parameters:
    EXTERNAL: 'TRUE'
    classification: json
    compressionType: gzip             # Compression information is here
    inputformat: org.apache.hadoop.mapred.TextInputFormat
    location: s3://jt-eu-west-1-dev-datalake-tracking-stage-bucket/jobteaser_job_ad_display
    outputformat: org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat
    serde.param.serialization.format: '1'
    serde.serialization.lib: org.openx.data.jsonserde.JsonSerDe
    transient_lastDdlTime: '1631627716'
  PartitionKeys:
  - Name: dt
    Type: string
  TableType: EXTERNAL_TABLE
```
And with a table storing it's data as Parquet:
```yaml
TableMetadata:
  Column: []
  CreateTime: null
  LastAccessTime: null
  Name: parquet_table
  Parameters:
    EXTERNAL: 'TRUE'
    inputformat: org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat
    location: s3://jt-eu-west-1-dev-datalake-tracking-datalake-bucket/jobteaser_job_ad_display
    outputformat: org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat
    parquet.compression: snappy              # Compression information is here.
    serde.param.serialization.format: '1'
    serde.serialization.lib: org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe
    transient_lastDdlTime: '1631628025'
  PartitionKeys:
  - Name: dt
    Type: string
  TableType: EXTERNAL_TABLE
```
There are still a lot I need to learn to be confident I can implement this correctly. Of course, if you have a clear picture of what the implementation should be, I'd be more than happy to fast track that particular feature.

Reply:
For now I put this [over there](https://github.com/laughingman7743/PyAthena/pull/264/files#diff-0cc6c6cef67adf7b211eccc2704d5a9d298a4303d474e5fa10a83fcc49c4447eR453) but it will move in the next PR, where I'll add support for other file formats. So, please, bear with me.

Reply:
No, it should be placed in a property of the model class.